src/components/tag/tag.styles.ts

```diff
@@ -75,7 +75,7 @@ export default css`
 
   .tag--small {
     font-size: var(--sl-button-font-size-medium);
-    height: calc(var(--sl-input-height-small) * 0.9);
+    height: calc(var(--sl-input-height-small) * 0.85715);
```

Comment:
We might want to make sure these render OK in different browsers / platforms. I feel like some of them are pickier about how they render odd font sizes.

Reply:
Ah, great call. What do you think if I just did away with the `calc` function entirely and just add a variable for the height that we know we want these to be? Feels more straightforward? If that sounds good to you, I'll update.

Reply:
See my comment below